tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "target": "ES2019", // Node.js v12
+    "module": "commonjs", // Node.js
+    "outDir": "dist",
+    "removeComments": true,
+    "strict": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true
+  }
+}
```

Comment:
Some recommendations:  

```json
"noImplicitAny": true,  
"noImplicitReturns": true,  
"noUnusedParameters": true,  
"strictNullChecks": true,  
"skipLibCheck": true,  
"skipDefaultLibCheck": true,
```

Reply:
`noImplicitAny` and `strictNullChecks` are enabled by default once `strict` is true.
I'm going to add the other ones.
Thanks!